Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Latest version of ubuntu
+FROM ubuntu:latest
+
+# Default git repository
+ENV GIT_REPOSITORY https://github.com/TheFox/flickr-cli.git
+
+# Innstall git, php + other tools, the the app and remove the unneded apps
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y php-bcmath php-curl php-simplexml composer git unzip \
+    && git clone $GIT_REPOSITORY \
```

Comment:
Why clone the repo again, when it's already accessible locally?

If we're already suggesting people mount their flickr-cli directory as `/mnt`, could we do away with this clone, and change the entrypoint php arg to `/mnt/application.php` instead?

This would mean that the Docker set up could be used for people developing flickr-cli as well.